src/notation/qml/MuseScore/NotationScene/internal/PercussionPanelToolBar.qml

```diff
@@ -206,7 +206,7 @@ Item {
         navigation.row: 0
 
         onClicked: {
-            root.model.finishEditing()
+            root.model.finishEditing(/*discardChanges*/ false)
```

Comment:
Perhaps we should assign false by default so that we don't always have to specify it explicitly